mitosheet/src/components/taskpanes/AITransformation/AITransformationResultSection.tsx

```diff
@@ -64,7 +67,7 @@ const AITransformationResultSection = (props: AITransformationResultSectionProps
                                 }
                             })
                         }}>
-                        <span>Created:</span> <span className="text-underline">{dfName}</span>  ({numRows} rows, {numColumns} columns)
+                        <span className={classNames({'text-color-recon-created': props.isMostRecentResult})}>Created:</span> <span className="text-underline">{dfName}</span>  ({numRows} rows, {numColumns} columns)
```

Comment:
I think we can always highlight this text, no?

Reply:
I wanted to only highlight in the last one so it was clear what is highlighted in the sheet.